code.py

```
@@ -1 +1,130 @@
+    return normalized
+
+
+@mcp.tool
+def get_screenshot(region: Optional[Sequence[int]] = None) -> str:
+    """Return a PNG screenshot encoded as a data URL.
+
+    If ``region`` is provided, it must contain ``[x, y, width, height]`` values.
+    """
+
+    if region is not None and len(region) != 4:
+        raise ValueError("Region must contain exactly four integers: x, y, width, height.")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)